shared/dev/dev-menu.desktop.js

```diff
@@ -1,18 +1,12 @@
 // @flow
-import * as LoginGen from '../actions/login-gen'
 import * as React from 'react'
-import engine from '../engine'
 import {BackButton, Box, Text} from '../common-adapters'
 import {connect, type Dispatch} from '../util/container'
 import {globalStyles, globalColors} from '../styles'
-import {navigateAppend, navigateUp} from '../actions/route-tree'
+import {navigateUp} from '../actions/route-tree'
 
 function DevMenu(props) {
-  const menuItems = [
-    {name: 'Dumb components', hasChildren: true, onClick: props.onDumbSheet},
```

Comment:
no dumb sheet and sign out already exists IRL so lets just remove this all
no one uses reset engine